projects/vanilla/src/styles/app.scss

```diff
@@ -64,8 +63,41 @@ $action-menu-breakpoint: sm;
 
 // default sinequa configuration
 @import 'themes/default';
-// allow cusomization
+
+body.sinequa:not(.dark) {
+    @import "../../../../node_modules/@sinequa/components/theme/vanilla";
+}
+
 @import 'themes/custom';
+// allow cusomization
+// @import 'themes/cyberpunk';
+
 
+// Facets -- TO REMOVE WHEN CSS WILL BE IN THE COMPONENT
+sq-facet-card {
```

Comment:
I think it can be removed if you update the version of the sinequa libraries in the package.json ?

Reply:
I'will do this

Reply:
removed